tests/garage/tf/policies/test_gaussian_mlp_policy_with_model.py

```diff
@@ -0,0 +1,173 @@
+        objective1 = tf.reduce_mean(kl_diff_sym1)
+
+        kl_func = tensor_utils.compile_function([obs_ph], [objective1])
+        kl_func(obs, obs)  # result
+
+        # * New way *
+        # equvaient to distribution.kl_sym()
+        kl_diff_sym2 = tfp.distributions.kl_divergence(
+            policy3.model.networks['default'].distribution,
+            policy4.model.networks['default'].distribution)
+        objective2 = tf.reduce_mean(kl_diff_sym2)
+
+        self.sess.run(   # result
+            [objective2],
+            feed_dict={
+                policy3.model.networks['default'].input: obs,
+                policy4.model.networks['default'].input: obs
+            })
+
+        ############################################################
+        # log_likelihood_sym
+        # * Existing way *
+        log_prob_sym1 = policy1.distribution.log_likelihood_sym(
+            obs_ph, dist2_sym)
+        log_prob_func = tensor_utils.compile_function([obs_ph], log_prob_sym1)
+        log_prob_func(obs)  # result
+
+        # * New way *
+        # equvaient to distribution.log_likelihood_sym(X, dist)
+        log_prob_sym2 = policy3.model.networks[
+            'default'].distribution.log_prob(
+                policy3.model.networks['default'].input)
+        self.sess.run(  # result
+            log_prob_sym2,
+            feed_dict={policy3.model.networks['default'].input: obs})
+
+        ############################################################
+        # entropy_sym
+        # * Existing way *
+        entropy_sym1 = policy1.distribution.entropy_sym(
+            dist1_sym, name='entropy_sym1')
+        entropy_func = tensor_utils.compile_function([obs_ph], entropy_sym1)
+        entropy_func(obs)  # result
+
+        # * New way *
+        # equvaient to garage.tf.distribution.entropy_sym(dist)
+        entropy_sym2 = policy3.model.networks['default'].distribution.entropy()
+        self.sess.run(  # result
+            entropy_sym2,
+            feed_dict={policy3.model.networks['default'].input: obs})
+
+        ############################################################
+        # likelihood_ratio_sym
+        # * Existing way *
+        likelihood_ratio_sym1 = policy1.distribution.likelihood_ratio_sym(
+            obs_ph, dist1_sym, dist2_sym, name="li_ratio_sym1")
+        likelihood_ratio_func = tensor_utils.compile_function(
+            [obs_ph], likelihood_ratio_sym1)
+        likelihood_ratio_func(obs)  # result
+
+        # * New way *
+        # tf.distributions seems doesn't have this available
+        # maybe do it ourselves
+        with tf.name_scope('li_ratio_sym2'):
+            log_prob_diff = policy3.model.networks[
+                'default'].distribution.log_prob(
+                    obs_ph, name='log_prob_obs') - policy4.model.networks[
+                        'default'].distribution.log_prob(obs_ph)
+
+            likelihood_ratio_sym2 = tf.exp(log_prob_diff)
```

Comment:
Why `tf.exp`?

Reply:
I was referring to the `likelihood_ratio_sym` in garage.
```python
    def likelihood_ratio_sym(self,
                             x_var,
                             old_dist_info_vars,
                             new_dist_info_vars,
                             name=None):
        with tf.name_scope(name, "likelihood_ratio_sym",
                           [x_var, old_dist_info_vars, new_dist_info_vars]):
            logli_new = self.log_likelihood_sym(x_var, new_dist_info_vars)
            logli_old = self.log_likelihood_sym(x_var, old_dist_info_vars)
            return tf.exp(logli_new - logli_old)
```

Reply:
Isn't this function in tfp is the [log_likelihood](https://www.tensorflow.org/probability/api_docs/python/tfp/distributions/Distribution#log_prob)?

Reply:
You should add this interface in policy. This is also a trivial issue.